lib/services/local/users.go

```diff
@@ -1420,17 +1424,34 @@ func (s *IdentityService) CreateAssistantConversation(ctx context.Context, usern
 	return &proto.CreateAssistantConversationResponse{Id: conversationID}, nil
 }
 
+func (s *IdentityService) getConversation(ctx context.Context, username, conversationID string) (*Conversation, error) {
+	item, err := s.Get(ctx, backend.Key(assistantConversationPrefix, username, conversationID))
```

Comment:
do we need to sanitize this considering it's a user provided variable that suspiciously isn't passed as a `uuid.UUID` for some (?) reason?

Reply:
I'll address that while backporting to master.